spec/index.bs

```diff
@@ -4372,6 +4391,28 @@ The {{GPUTextureUsage}} flags determine how a {{GPUTexture}} may be used after i
                 1. Set |t|.{{GPUTexture/dimension}} to |descriptor|.{{GPUTextureDescriptor/dimension}}.
                 1. Set |t|.{{GPUTexture/format}} to |descriptor|.{{GPUTextureDescriptor/format}}.
                 1. Set |t|.{{GPUTexture/usage}} to |descriptor|.{{GPUTextureDescriptor/usage}}.
+                1. If |t|.{{GPUObjectBase/[[device]]}}.{{device/[[features]]}} does not [=list/contain=] {{GPUFeatureName/"core-features-and-limits"}}
+                    1. If |descriptor|.{{GPUTextureDescriptor/textureBindingViewDimension}} is [=map/exist|provided=]:
+                        1. Set |t|.{{GPUTexture/textureBindingViewDimension}} to |descriptor|.{{GPUTextureDescriptor/textureBindingViewDimension}}.
+                    1. Otherwise, if |descriptor|.{{GPUTexture/dimension}} is:
```

Comment:
oops one more nit: I prefer the "if" and "else" parts to be in the same numbered step. Will push a commit that does this and also changes the indentation accordingly